boards/pinetime/include/board.h

```diff
@@ -66,6 +67,24 @@ extern "C" {
 #define ILI9341_PARAM_NUM_LINES    240U
 /** @} */
 
+/**
+ * @name PineTime NOR flash hardware configuration
+ */
+/** @{ */
```

Comment:
(style nick: why seperate comment block? Maybe do as the others (for consistency).)

Reply:
Do as the other config blocks or do as the other [boards](https://github.com/RIOT-OS/RIOT/blob/master/boards/mulle/include/board.h#L147-L161)?

Reply:
It seems that mulle is the board exception:
```
git grep "/\*\* @{ \*/" boards/
boards/mulle/include/board.h:/** @{ */
boards/mulle/include/board.h:/** @{ */
boards/mulle/include/board.h:/** @{ */
boards/mulle/include/board.h:/** @{ */
boards/mulle/include/board.h:/** @{ */
```

Reply:
Never mind, I misread @kaspar030's comment here. I took the liberty to amend the change directly.